src/main/java/org/apache/mesos/hdfs/config/HdfsFrameworkConfig.java

```diff
@@ -188,7 +236,7 @@ public String getFrameworkHostAddress() {
       try {
         hostAddress = InetAddress.getLocalHost().getHostAddress();
       } catch (UnknownHostException e) {
-        throw new RuntimeException(e);
+        log.error("issue with host address", e);
```

Comment:
Throw exception, return null? Let's make sure the caller(s) handle this potential error.